src/node_config.cc

```diff
@@ -28,10 +28,21 @@ using v8::ReadOnly;
 void InitConfig(Local<Object> target,
                 Local<Value> unused,
                 Local<Context> context) {
-#ifdef NODE_FIPS_MODE
   Environment* env = Environment::GetCurrent(context);
```

Comment:
Take a look immediately below... the `Environment* env` was only being declared if `NODE_FIPS_MODE` was set. This just pulled that one line out of that block.